content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/log-in-chainguard-enforce/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Signing into Chainguard Enforce"
+type: "article"
+description: "How to log into Chainguard Enforce for Kubernetes"
+date: 2022-11-30T11:07:52+02:00
+lastmod: 2022-11-30T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 500
+toc: true
+---
+
+> _This document relates to Chainguard Enforce, which you will need access to in order to follow along. You can request access through selecting **Chainguard Enforce for Kubernetes** on our [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)._
+
+Chainguard Enforce currently offers login and authentication via three different OpenID Connect identity providers: Google, GitHub, and GitLab. [OpenID Connect](https://openid.net/connect/) (OIDC) provides an identity layer on top of the OAuth 2.0 protocol, and it is integrated with both the Chainguard Enforce Console web browser interface, and `chainctl` on the command line. This document will walk through both of these workflows.
+
+## Signing in through the Chainguard Enforce Console
+
+Once you have access to Chainguard Enforce, you can navigate a web browser to [console.enforce.dev](https://console.enforce.dev). A sign in page will be displayed, similar to the one below. 
+
+![Chainguard Enforce Console sign in page](console-sign-in.png)
+
+From here, you have three options to authenticate into the Console via one of the following OIDC providers: 
+
+* [Google](https://developers.google.com/identity/openid-connect/openid-connect)
+* GitHub
```

Comment:
I couldn't find a similar link for GitHub, only their guidance on authenticating via cloud providers for GitHub Actions 😢

Reply:
This is the only official GitHub doc I could find about OIDC: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers